src/System.Runtime.Environment/src/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" />
```

Comment:
@FiveTimesTheFun : I am pushing this to future branch, and eric's commit is not in future yet. @ellismg : Should this go in future or master?